27.md

```diff
@@ -1,12 +1,51 @@
+
+Additionally, this project aims to advance [EDAM](https://edamontology.org/page) as a standard, by applying it to a large genomic data science software/data ecosystem. By extending the EDAM ontology and the processes available through the RSEc to cover these Bioconductor packages, and designing automated mechanisms for synchronising descriptions between Bioconductor and the ELIXIR RSEc, we will significantly improve the search and discovery process for users, and strengthen the bioinformatics research infrastructure.
+
+This project will kick-start a long-term mutually beneficial collaboration between the ELIXIR Tools Platform and the Bioconductor community. 
+
+:dart: **Short-term BioHackathon goals:**
+
+* Mapping of EDAM and biocViews terms
+* “Gold standard" manual annotation of a subset of Bioconductor packages in bio.tools
+* Assessing development or adaptation of a tool for automated EDAM suggestions from biocViews or package content
+
+:dart: **Long-term goals:**
+
+* Extend EDAM to all Bioconductor software packages, and also the thousands of Bioconductor annotation and experiment resources
+* Phase-out biocViews for systematic EDAM annotation
+* Synchronise Bioconductor packages with bio.tools (via automated integration with ELIXIR RSEc)
+
+## Interested in contributing?
+
+:loudspeaker: Reach out! 
+
+* [Bioconductor slack community](https://community-bioc.slack.com/): **#edam-collaboration** 
+* [ELIXIR Europe slack community](https://elixir-europe.slack.com/): **#edam_ontology** 
+
+Our project is committed to inclusivity, guided by the [Bioconductor Code of Conduct](https://www.bioconductor.org/about/code-of-conduct), as well as the [ELIXIR code of conduct for events](https://elixir-europe.org/events/code-of-conduct) and the [ELIXIR RSEc code of conduct](https://github.com/research-software-ecosystem/content/blob/master/CODE_OF_CONDUCT.md). We value inputs from different perspectives - from ontology experts to developers to end user experience - across a diversity of professional, personal, cultural, or linguistic backgrounds.
+
+Remote participation is welcome, and would ideally be planned with the interested parties ahead of the event, to ensure we can get the necessary setup ready to work during the event and have everybody able to fully enjoy and take advantage of the event.
+
+## Project abstract
```

Comment:
Proposing to keep the heading the same so that it matches the rest of the projects

```suggestion
## Abstract
```